.gitignore

```diff
@@ -18,10 +18,12 @@ Reinterop.deps.json
 Reinterop.deps.json.meta
 Reinterop.pdb
 Reinterop.pdb.meta
-CesiumForUnityNative-*.dll
-CesiumForUnityNative-*.dll.meta
-libCesiumForUnityNative-*.so
-libCesiumForUnityNative-*.so.meta
-libCesiumForUnityNative-*.dylib
-libCesiumForUnityNative-*.dylib.meta
+CesiumForUnityNative.dll
+CesiumForUnityNative.dll.meta
+libCesiumForUnityNative.so
+libCesiumForUnityNative.so.meta
+libCesiumForUnityNative.dylib
+libCesiumForUnityNative.dylib.meta
 Documentation~/Reference/
+generated~/
+test-server~/
```

Comment:
How might these files get generated?

Reply:
I often use `generated~` as the Reinterop output directory, i.e., when compiling with `-generatedfilesout` (see [here](https://github.com/CesiumGS/cesium-unity/blob/main/Reinterop~/README.md#temporarily-disabling-the-code-generator)) to look at the generated code.

`test-server~` was where I stored a node.js server I used to test Unity on the web.

No good reason I should have committed either, though. I'll remove them.